### DISCUSSION TITLE
Core WebRTC Plugin Broken?

### DISCUSSION THREAD

**Discussion by carlosnasillo** [koush/scrypted#1212] [Q&A] [answered]
The core WebRTC plugin seems to have the option under "Providing Things" to add a new camera (ie. `Add New` button), however there does not seem to be any functionality behind it. Once you click and create a new camera name, there are no options to configure an incoming stream (ie. enter a webrtc/http URL)

Is it currently broken, or is the intended functionality other than to import WebRTC feeds into scrypted? Thanks

**Comment by koush** [comment, marked as answer]
https://github.com/koush/scrypted/discussions/1145#discussioncomment-7360564

**Reply by carlosnasillo** [reply]
Thanks, I had actually seen that thread, however my question was about using a WebRTC feed as a camera source **into Scrypted**, rather than as a rebroadcasted feed from Scrypted - but I guess the answer would be the same.

**Reply by koush** [reply]
correct
